docs/maui/essentials/apptheme-resources.md

```diff
@@ -0,0 +1,119 @@
+
+```xaml
+<ContentPage
+    x:Class="CommunityToolkit.Maui.Sample.Pages.MyPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit">
+    <ContentPage.Resources>
+        <toolkit:AppThemeResource Light="dark.png" Dark="light.png" x:Key="MyImageSource" />
+    </ContentPage.Resources>
+
+    <VerticalStackLayout>
+        <Image Source="{toolkit:ThemeResource ImageSource}" />
+    </VerticalStackLayout>
+</ContentPage>
+```
+
+#### AppThemeColor
+
+The `AppThemeColor` is a specialized theme-aware [`Color`](xref:Microsoft.Maui.Graphics.Color) that allows you to set a `Color` for the `Light`, `Dark` and `Default` properties.
+
+The following example shows how to use `AppThemeColor` through a `ResourceDictionary`:
+
+```xaml
+<ContentPage
+    x:Class="CommunityToolkit.Maui.Sample.Pages.MyPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit">
+    <ContentPage.Resources>
+        <toolkit:AppThemeColor Light="Red" Dark="Green" x:Key="LabelTextColor" />
+    </ContentPage.Resources>
+
+    <VerticalStackLayout>
+        <Label TextColor="{toolkit:ThemeResource LabelTextColor}" />
+    </VerticalStackLayout>
+</ContentPage>
+```
+
+#### Consuming AppThemeColor and AppThemeResource Through Styles
+
+Because we can use these theme-aware resources in a `ResourceDictionary`, that means we can also consume them through a `Style`.
+
+The following example shows how to use `AppThemeColor` through a `Style`:
+
+```xaml
+<ContentPage
+    x:Class="CommunityToolkit.Maui.Sample.Pages.MyPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit">
+    <ContentPage.Resources>
+        <toolkit:AppThemeColor Light="Red" Dark="Green" x:Key="LabelTextColor" />
+
+        <Style x:Key="Headline" TargetType="Label">
+            <Setter Property="FontFamily" Value="Segoe UI" />
+            <Setter Property="FontSize" Value="10" />
+            <Setter Property="TextColor" Value="{toolkit:ThemeResource LabelTextColor}" />
+        </Style>
+    </ContentPage.Resources>
+
+    <VerticalStackLayout>
+        <Label Style="{StaticResource Headline}" />
+    </VerticalStackLayout>
+</ContentPage>
+```
+
+## Extensibility
+
+Both `AppThemeResource` and `AppThemeColor` inherit from the abstract class `AppThemeObject<T>`. If you have a need for a more strongly typed resource that is not available in the .NET MAUI Community Toolkit, you can create your own inheritance.
+
+## Properties
+
+THe below table describes the properties for `AppThemeResource` and `AppThemeColor`. For `AppThemeColor`, the types of each property will be `Color` instead of `object`.
```

Comment:
```suggestion
The below table describes the properties for `AppThemeObject` and `AppThemeColor`. For `AppThemeColor`, the types of each property will be `Color` instead of `object`.
```